### PR TITLE
Create simple, dockerized argo server Fixed #8

### LINKED ISSUE #8: Create simple, dockerized argo server
As it's in the title, create borsuk server for argo project.

### PR DESCRIPTION
**Pull request by Agri**

### CODE REVIEW

File: server/borsukStart.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from flask import Flask
```
Comment (klangner) [inline server/borsukStart.py]: It is better to name this file to "main.py"

File: server/borsukStart.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+app = Flask(__name__)
```
Comment (klangner) [inline server/borsukStart.py]: The name of the application should be "Argo"

File: server/borsukStart.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+@app.route("/first")
```
Comment (klangner) [inline server/borsukStart.py]: Please remove all warnings from this file

**Comment by Agri** [comment]
@klangner 
Changed file name. 
I tried to change project name, bu the name provided in the constructor of the flask is internally used to set bunch of options.
Spyder's code analysis, even run explicitly, does not show any problems

**Comment by Agri** [comment]
added lines before every function

**Pull request merged**